Recruit-Training/Advanced-Recruit-Training/Embedded-Teaser/Training/Test/TrainingTest.c

```diff
@@ -4,25 +4,48 @@
 
 void runTrainingTests()
 {
-    // Run all the tests here using RUN_TEST()
+	RUN_TEST(test_EverythingValid);
+	RUN_TEST(test_EverythingInvalid);
+	RUN_TEST(test_OnlyLightsInvalid);
+	RUN_TEST(test_OnlyMotorsInvalid);
 }
 
 void test_EverythingValid()
 {
-
+	printf("\nTesting trainingTask - Should Pass\n");
+	uint8_t inputData[3] = {0b11001011, 0b11010111, 0b00001010}; 
+	trainingTask (inputData);
+	TEST_ASSERT_EQUAL_UINT8_ARRAY (inputData, outputArray, 3);
```

Comment:
Not really necessary, but it would be a good idea to use the `_MESSAGE` versions of the test macro. The only difference is that there's one more argument that is a string for the message to be printed in case of a failure